Classes/DDFileLogger.m

```diff
@@ -433,6 +434,28 @@ - (NSString *)newLogFileName {
     return [NSString stringWithFormat:@"%@ %@.log", appName, formattedDate];
 }
 
+- (void)modifyLogFileCreateDateIfNeed:(NSString *)filePath
+{
+    NSDate* fileCreateDate = [NSDate date];
+    if (_lastLogFileCreateDate == nil) {
+        DDLogFileInfo *logFileInfo = [self sortedLogFileInfos].firstObject;
+        if (logFileInfo) {
+            _lastLogFileCreateDate = logFileInfo.creationDate;
+        }
+    }
+    if (_lastLogFileCreateDate && (int)_lastLogFileCreateDate.timeIntervalSince1970 >= (int)fileCreateDate.timeIntervalSince1970) {
+        fileCreateDate = [_lastLogFileCreateDate dateByAddingTimeInterval:1];
```

Comment:
Each time a file is created, will be compared with _lastLogFileCreateDate, fileCreateDate always larger than _lastLogFileCreateDate. Finally fileCreateDate assigned to _lastLogFileCreateDate. Why also collision?